<launch>
  <node pkg="gopigo3_slam" type="waypoints_goal_params.py" name="movebase_client_waypoints" output="screen" >
    <rosparam file="$(find gopigo3_slam)/config/waypoints.yaml" command="load" />
  </node>
</launch>